### ISSUE TITLE
 Sorting does not work if the media is moved to two lines

### ISSUE THREAD

**Issue by movie4** [OlehDutchenko/sort-css-media-queries#6]
Sorting does not work if the media is moved to two lines

```
@media (min-width: 48em)
  and (max-width: 59.999em) {
  body {
    border: solid 10px blue;
  }
}

@media (min-width: 40em)
  and (max-width: 47.999em) {

  body {
    border: solid 10px black;
  }
}

@media (min-width: 15em)
  and (max-width: 47.999em) {
  body {
    border: solid 10px red;
  }
}
```

if input 

```
@media (min-width: 48em) and (max-width: 59.999em) {
  body {
    border: solid 10px blue;
  }
}

@media (min-width: 40em)  and (max-width: 47.999em) {

  body {
    border: solid 10px black;
  }
}

@media (min-width: 15em)  and (max-width: 47.999em) {
  body {
    border: solid 10px red;
  }
}
```

work correct

```
@media (min-width: 15em) and (max-width: 47.999em) {
  body {
    border: solid 10px red
  }
}

@media (min-width: 40em) and (max-width: 47.999em) {
  body {
    border: solid 10px black
  }
}

@media (min-width: 48em) and (max-width: 59.999em) {
  body {
    border: solid 10px blue
  }
}
```

**Comment by OlehDutchenko** [comment]
Hello @movie4! Thanks for reporting.
Already published new patch version on npm `1.5.4`

**Comment by movie4** [comment]
@dutchenkoOleg — thanks a lot 👍
